x/wasm/types/wasmer_engine.go

```diff
@@ -82,7 +82,7 @@ type WasmerEngine interface {
 	// replace it. This allows it to run a migration step if needed, or return an error if unable to migrate
 	// the given data.
 	//
-	// MigrateMsg has some data on how to perform the migration.
+	// Msg has some data on how to perform the migration.
```

Comment:
This propably refers to the argument `migrateMsg` which is not renamed. Let's use the variable name here in text, no matter which one we use.

Reply:
Over aggressive renaming.

Should I rename the variables in this interface as well? I guess it has nothing to do with proto and is non API-breaking, so we can do it later.

Reply:
I don't mind. I don't even understand why this file exist and those signatures are not imported from wasmvm.

Reply:
This is an interface that matches wasmvm (the implementation), but allows him to swap out the vm with a mock that let's us test mock contracts in Go. This makes for much simpler unit testing of the higher level logic (eg. go contract that returns such and such events in reply, then see how the are assembled through the keeper)